xorg: update xorg section (#38)
void-linux/void-docs: src/config/xorg/index.md

# Xorg

This section details the manual installation and configuration of the Xorg
display server and common related services and utilities. If you would just like
to install a full desktop environment, it is recommended to try one of the
[flavor images](../../installation/live-images/index.md#flavor-images)

## Installation

Void provides a comprehensive `xorg` package which installs the server and all
of the free video drivers, fonts, and base applications. This package is a safe
option, and should be adequate for most systems which don't require proprietary
video drivers.

If you would like to select only the packages you need, the `xorg-minimal`

> package contains the base xorg server *only*. Remember that you will likely need

Reviewer: ```suggestion
package contains the base xorg server *only*. If you install only `xorg-minimal`, you will likely need
```
I think it's better to be specific here.

to install a font package (like `xorg-fonts`), a terminal emulator (like
`xterm`), and a window manager to have a usable graphics system.

> ## Drivers

Reviewer: I'd add a short sentence (or two, if more needs to be said) here to lead into the sub-headings. Something like, "Void provides both open-source and proprietary (non-free) drivers."

### Open Source Drivers

The open source drivers are installed with the `xorg` package by default, or may
be installed individually.

| Brand  | Type        | Driver               |
|--------|-------------|----------------------|
| AMD    | Open Source | `xf86-video-amdgpu`  |
| ATI    | Open Source | `xf86-video-ati`     |
| Intel  | Open Source | `xf86-video-intel`   |
| NVIDIA | Open Source | `xf86-video-nouveau` |

> > Note: Some recommend not installing the `xf86-video-intel` driver for fourth

Author: > Note: Fourth generation intel users may want to use the default xorg driver,
> rather than installing `xf86-video-intel` driver package. For more
> information, see the [Arch wiki...

Is this better? I feel like it is worth mentioning. But a more neutral tone, sending them somewhere else is better.

> > Note: Some recommend not installing the `xf86-video-intel` driver for fourth

Reviewer: That sounds better to me.

# Xorg

## Installation

Void provides a comprehensive `xorg` package which installs the server and all
of the free video drivers, fonts, and base applications. This package is a safe
option, and should be adequate for most systems which don't require proprietary
video drivers.

If you would like to select only the packages you need, the `xorg-minimal`
package is provided, but remember you will need to install `xorg-fonts` and a
terminal emulator (like `xterm`) along with your window manager to have a
working graphics system.

## Drivers

### Open Source Drivers

The open source drivers are installed with the `xorg` package by default, or may
be installed individually.

| Brand  | Type        | Driver               |
|--------|-------------|----------------------|
| AMD    | Open Source | `xf86-video-amdgpu`  |
| ATI    | Open Source | `xf86-video-ati`     |
| Intel  | Open Source | `xf86-video-intel`   |
| NVIDIA | Open Source | `xf86-video-nouveau` |

> 

Reviewer 2: These are all optional DDX/2D acceleration drivers. When they're not found on the system, Xorg will fallback to the generic modesetting/GLAMOUR driver. It seems like it's generally recommended to install these instead of using modesetting/GLAMOUR, except in the case of Intel, where its recommended to try modesetting/GLAMOUR first, and to only install `xf86-video-intel` if you have problems ([arch wiki reference](https://wiki.archlinux.org/index.php/Intel_graphics#Installation))

To install 3D acceleration  drivers, you actually require the `libGL` and `mesa-*-dri` packages instead.

Author: I'm not at all an expert on this. I generally try things for exactly as long as xorg is not working ;-) Thanks for the info!

Author: Ok, I mentioned the intel issue, with a link to the arch wiki. It looks like the xf86-video-* packages install the appropriate mesa-*-dri packages so I don't want to document that until a reason comes up.
